alerter/src/main/java/org/apache/hertzbeat/alert/service/impl/AlertDefineServiceImpl.java

```diff
@@ -158,6 +164,18 @@ public AlertDefine getMonitorBindAlertAvaDefine(long monitorId, String app, Stri
 
     @Override
     public Page<AlertDefine> getAlertDefines(List<Long> defineIds, String search, Byte priority, String sort, String order, int pageIndex, int pageSize) {
+        // parse translation content list
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<String> searchList = Collections.emptyList();
+        if (org.apache.commons.lang3.StringUtils.isNotEmpty(search)) {
```

Comment:
change all "org.springframework.util.StringUtils" to  "org.apache.commons.lang3.StringUtils", and import org.apache.commons.lang3.StringUtils

Reply:
LGTM